data/version.yml

```diff
@@ -1,3 +1,36 @@
+0.16.6:
+  release_date: "2016/06/14"
+  git_commit: ""
```

Comment:
I actually had a commit hash in the previous commit, but it was not the final one so removed it and force-pushed :)

I'll make amends during the release process.